frontend/public/components/monitoring.tsx

```diff
@@ -907,6 +928,69 @@ const CreateSilence = () => {
     : <SilenceForm defaults={{matchers}} saveButtonText="Create" title="Silence Alert" />;
 };
 
+const MetricsList = ({metrics}) => <div className="co-m-table-grid co-m-table-grid--bordered">
+  <div className="row co-m-table-grid__head">
+    <div className="col-xs-9 query-browser-metric__wrapper">
+      <div className="query-browser-metric__color"></div>
+      SERIES
+    </div>
+    <div className="col-xs-3">VALUE</div>
+  </div>
+  <div className="co-m-table-grid__body">
+    {_.map(metrics, (m, i) => <div className="row" key={i}>
+      <div className="col-xs-9 query-browser-metric__wrapper">
+        <div className="query-browser-metric__color" style={{backgroundColor: graphColors[parseInt(i, 10) % graphColors.length]}}></div>
+        <div className="query-browser-metric__labels">
+          {_.map(m.labels, (v, k) => `${k}="${v}",`).join('').replace(/,$/, '')}
+        </div>
+      </div>
+      <div className="col-xs-3">{m.value}</div>
+    </div>)}
+  </div>
+</div>;
+
+const QueryBrowserPage = () => {
+  const [metrics, setMetrics] = React.useState(getURLSearchParams().query);
```

Comment:
This looks like it's the wrong initial value?

Reply:
Oops